src/shared/components/case-editor/services/cases.service.ts

```diff
@@ -297,13 +297,16 @@ export class CasesService {
   }
 
   private processTasksOnSuccess(caseData: any, eventData: any): void {
-    this.workAllocationService.completeAppropriateTask(caseData.id, eventData.id)
-      .subscribe(() => {
-        // Success. Do nothing.
-      }, error => {
-        // Show an appropriate warning about something that went wrong.
-        console.warn('Could not process tasks for this case event', error);
-      });
+    // This is used a feature toggle to 
+    // control the work allocation
+    if(this.appConfig.getWorkAllocationApiUrl()) {
```

Comment:
could this be refactored to make the comment redundant...

```
const workAllocationFeatureToggle = this.appConfig.getWorkAllocationApiUrl()

if (workAllocationFeatureToggle ) {...
```

Reply:
@phillip-whitaker Agree but, I want the comment to make it explicit to the developer that config needs a null which is used a feature toggle.

Reply:
@phillip-whitaker, whilst I don't disagree with putting it in a `const` first, it shouldn't have a name that implies that it's a feature toggle in and of itself, because it is indeed the URL to use.

The mechanism @udaydenduluri33 has put in place here is to avoid the need for _actual_ feature toggles in the CCD Case UI Toolkit and I think a comment explaining why this one behaves differently to the others is probably sensible. :)